### PR TITLE
[New] How to Install Cassandra (Debian, Ubuntu, CentOS)

### PR DESCRIPTION
**Pull request by leslitagordita**
* Add How to Install Cassandra, Ubuntu, Debian, and CentOS

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit fc3e2147be79ef74797bab9fcdbfa3d98b9a2748

https://deploy-preview-3116--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit dd0b955e55941c96f50b27fd64a7a0be78bcecc5

https://deploy-preview-3116--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/databases/cassandra/how-to-install-apache-cassandra/ubuntu/index.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+        gpg --export --armor 749D6EEC0353B12C | sudo apt-key add -
+
+        gpg --keyserver pgp.mit.edu --recv-keys A278B781FE4B2BDA
+        gpg --export --armor A278B781FE4B2BDA | sudo apt-key add -
+
+## Install Cassandra and Supporting Applications
+
+In this section, you will update your Linux system software and install Java along with Cassandra.
+
+1.  Install Cassandra, Java, and NTP. Answer "yes" to the Java installation prompts regarding the license terms (Ubuntu only). NTP will help keep the Cassandra node synced to the correct time.
```
Comment (andystevensname) [inline docs/databases/cassandra/how-to-install-apache-cassandra/ubuntu/index.md]: Does the "Ubuntu only" part mean there are terms for other distros in the license?

File: docs/databases/cassandra/how-to-install-apache-cassandra/ubuntu/index.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+1. From the Linux terminal (not cqlsh) clear the system cache. This command will not disturb your node's data.
+
+        nodetool flush system
+
+1.  Restart Cassandra:
+
+        sudo systemctl restart cassandra
+
+1. Log in with cqlsh and verify the new cluster name is visible.
```
Comment (andystevensname) [inline docs/databases/cassandra/how-to-install-apache-cassandra/ubuntu/index.md]: `cqlsh` should be in backticks.

File: docs/databases/cassandra/how-to-install-apache-cassandra/ubuntu/index.md [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+
+1. Type `exit` to return to the Linux command line.
+
+1.  Edit the `cassandra.yaml` file and replace the value in the `cluster_name` variable with the new cluster name you just set.
+
+        sudo vim /etc/cassandra/cassandra.yaml
+
+1.  Save and close.
+
+1. From the Linux terminal (not cqlsh) clear the system cache. This command will not disturb your node's data.
```
Comment (andystevensname) [inline docs/databases/cassandra/how-to-install-apache-cassandra/ubuntu/index.md]: `cqlsh` should be in backticks.

File: docs/databases/cassandra/how-to-install-apache-cassandra/ubuntu/index.md [hunk 4]
```diff
@@ -0,0 +1,299 @@
+---
+author:
+  name: Linode Community
+  email: docs@linode.com
+description: 'This guide presents instructions to deploy a scalable and development-driven NoSQL database with Apache Cassandra for both the Ubuntu 18.04.'
```
Comment (andystevensname) [inline docs/databases/cassandra/how-to-install-apache-cassandra/ubuntu/index.md]: "for both the" should be just "for"

File: docs/databases/cassandra/how-to-install-apache-cassandra/ubuntu/index.md (hunk 1 above)
Comment (rsyracuse) [reply docs/databases/cassandra/how-to-install-apache-cassandra/ubuntu/index.md]: This seems to be a standard prompt. Making an edit to reflect this

**Pull request merged**